modules/cpmso-yaml-provider-spec-openstack.adoc

```diff
@@ -45,4 +45,5 @@ providerSpec:
 <1> The secret name for the cluster. Do not change this value.
 <2> The {rh-openstack} flavor type for the control plane.
 <3> The {rh-openstack} cloud provider platform type. Do not change this value.
-<4> The control plane machines security group.
\ No newline at end of file
+<4> The control plane machines security group.
```

Comment:
> Not exactly sure what you want to know here.

Mostly just checking to make sure nothing looks weird/outdated :slightly_smiling_face: 

Looks like current docs cover `AvailabilityZone` and `RootVolume.AvailabilityZone` in the sample failure domain YAML. I don't see `RootVolume.VolumeType` anywhere, but I'm also a bit hesitant to dig in here since OSP is outside the scope of Cloud team for the most part. I will drop a note to the OSP writer though just for his awareness.